Update the README for Cohort 8 (#7)
bridge-school/personal-project-react: README.md

## Due Date
Due by 9am on August 12th

## Wireframes
For reference we have created an invision prototype to show you the flow of the react application which can be found here: https://bridgeschool.invisionapp.com/public/share/5GWTMSJMH

Please note that you do not need to style the application to match this prototype, it is purely to show you the flow of the application. Feel free to style the application as much (or as little) as you'd like.

## Requirements

> - Create a search form that allows you enter a github user name and submit it

Reviewer: ```suggestion
- Create a search form with an input that allows you enter a GitHub username and submit it
```

> - Create a search form with an input that allows you enter a GitHub username and submit it

Reviewer 2: ```suggestion
- Create a search form with an input that allows you enter a GitHub username and a button to search for the inputted GitHub username
```

> - Create a search form with an input that allows you enter a GitHub username and submit it

Author: @cwlsn thoughts on this small update?
```suggestion
- Create a search form with an input that allows you enter a GitHub username and button to fetch the user information for the inputted GitHub username
```

> - After submitting the form, you should make an API call to fetch the inputted username's information

Reviewer: Is the intention that they actually do an HTML submit, or more of a button onClick? If it's the latter it may help to not say "submit".

Author: I think for the scope of this we can just use a button onClick. Perhaps this is better wording?

```suggestion
- After clicking the search button, you should make an API call to fetch the inputted username's information
```

Reviewer: Agreeable
